pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```diff
@@ -142,10 +146,10 @@ a:hover {
   }
 
   .img-container {
-    background: url(/images/dance-hoc/dance_party_tutorial_hero.png);
```

Comment:
Weirdly another image was being drawn underneath...

Reply:
(Turns out this trick gets an image showing on mobile widths, filling the background while the image in front helps define its dimensions, so I'm going to leave it in after all.)